doc/ch-image.rst

```diff
@@ -93,13 +96,52 @@ Common options placed before or after the sub-command:
     subprocesses. Profile data should still be written on fatal errors, but
     not if the program crashes.
 
+  :code:`-q, --quiet`
+    Suppress logging info, can be repeated up to three times. Incompatible with
+    :code:`--verbose` and :code:`--debug`. You might use :code:`-q` if you want
+    to run a :code:`ch-image` command without seeing logging info, but you still
+    want to see output from subprocesses (e.g. results of the Dockerfile
+    :code:`RUN` instruction). You might use :code:`-qq` if you don’t want to see
+    any info chatter from :code:`ch-image` or its subprocesses. You might use
+    :code:`-qqq` if you want to suppress writing to stderr. A full breakdown of
+    the three levels can be seen in the table below.
+
+    .. list-table::
+      :header-rows: 1
+
+      * -
+        - :code:`-q`
+        - :code:`-qq`
+        - :code:`-qqq`
+      * - :code:`ch-image` stdout
+        - ✅
+        - ✅
+        - ✅
+      * - :code:`ch-image` fatal errors
+        - ✅
+        - ✅
+        - ✅
+      * - :code:`ch-image` warnings and errors
+        - ✅
+        - ✅
+        - ❌
+      * - :code:`ch-image` info logging
+        - ❌
+        - ❌
+        - ❌
+      * - Subprocess stderr
+        - ✅
+        - ✅
+        - ❌
+      * - Subprocess stdout
+        - ✅
+        - ❌
+        - ❌
```

Comment:
Couple of suggestions on this table:

1. Extend it to also include the default and the three `-v` levels (i.e., 7 columns total), and move it to its own section.
2. Order the rows from most suppressed to least suppressed.